### ISSUE TITLE
[multiple-field_type] lookup table has wonky sort order.

### ISSUE THREAD

**Issue by bloveless** [pyrocms/pyrocms#4319]
Created a team members stream that just has a name. Then I added all the names A-Z

<img width="1192" alt="screen shot 2017-04-17 at 6 44 14 pm" src="https://cloud.githubusercontent.com/assets/535135/25109988/1ccb89f2-239e-11e7-87aa-7069a47bb51c.png">

Created a departments stream that has a multiple field_type via lookup. Added some spaced out team members, 

<img width="1174" alt="screen shot 2017-04-17 at 6 44 39 pm" src="https://cloud.githubusercontent.com/assets/535135/25110007/3c37f474-239e-11e7-83b2-585e66f9373d.png">

Now if I add team members that originally are sorted in between the members that already exist they show up in odd places.

First I added A

<img width="1399" alt="screen shot 2017-04-17 at 6 44 50 pm" src="https://cloud.githubusercontent.com/assets/535135/25110030/63aafb8c-239e-11e7-96cc-43e056af3bb3.png">

And it showed up at the top

<img width="1177" alt="screen shot 2017-04-17 at 6 44 56 pm" src="https://cloud.githubusercontent.com/assets/535135/25110036/6b0739e0-239e-11e7-97ab-bde651806928.png">

Then I added D 

<img width="1398" alt="screen shot 2017-04-17 at 6 45 02 pm" src="https://cloud.githubusercontent.com/assets/535135/25110042/737a3fc8-239e-11e7-855c-bfddb25d1723.png">

And it shows up in the middle

<img width="1187" alt="screen shot 2017-04-17 at 6 45 09 pm" src="https://cloud.githubusercontent.com/assets/535135/25110046/7ae34458-239e-11e7-9c34-58ad9c02e858.png">

I would expect A and D to be added to the end of the list and then sorting to continue as expected.

One side note. If i push save without changing the sort order then the new entries show up at the bottom like I would expect.

<img width="841" alt="screen shot 2017-04-17 at 6 49 09 pm" src="https://cloud.githubusercontent.com/assets/535135/25110073/9e978486-239e-11e7-94d7-42361411c534.png">

If I manually adjust the sort order before saving then they seem to show up in the correct place.

**Comment by RyanThompson** [comment]
You've tested this on a clean install as well ya?

**Comment by bloveless** [comment]
I basically ran
composer create-project pyrocms/pyrocms
php artisan make:addon alc.module.team
php artisan make:stream team_members alc.module.team
php artisan make:stream departments alc.module.team

And setup the migrations.

TLDR;
Yes ;)

**Comment by Piterden** [comment]
Suppose, it should be just `members`
